manifests/quick-start-minimal.yaml

```diff
@@ -731,6 +731,10 @@ data:
     path: /metrics
     port: 9090
   namespaceParallelism: "10"
+  retentionPolicy: |
+    completed: 10
+    failed: 3
+    errored: 3
```

Comment:
I don't think this should be added by default? These should probably be removed here.

Reply:
Ok, will do.